feature/settings/src/main/java/com/google/jetpackcamera/settings/SettingsUiState.kt

```diff
@@ -167,8 +172,18 @@ sealed interface StabilizationUiState {
     // Stabilization selection completely disabled. Cannot open dialog.
     data class Disabled(val disabledRationale: DisabledRationale) : StabilizationUiState
 }
+// ////////////////////////////////////////////////////////////
+//
+// Settings that DON'T currently depend on constraints
```

Comment:
Would the audio permission be considered a constraint here?

Reply:
🫣 yes it should be be